docs/en/advanced/Security.md

```diff
@@ -349,5 +360,5 @@ respective under file system, such as HDFS transparent encryption or Linux disk
 
 ## Deployment
 
-It is recommended to start Alluxio master and workers under the same user.
+It is recommended to start Alluxio master and workers using the same operating system user.
```

Comment:
Can we just enforce this as a requirement? Many scripts like 'alluxio-start.sh all' won't work if the OS user differs. I believe our SecondaryMaster health check also relies on SSH and can't specify the OS user of the secondaries.